src/strongbox/core.clj

```diff
@@ -747,8 +759,9 @@
   [addon-summary]
   (binding [http/*cache* (cache)]
     (let [game-track (get-game-track) ;; scope creep, but it fits so nicely
+          strict? (get-game-track-strictness)
```

Comment:
get-game-track and get-game-track-strictness are barely wrappers for `addon-dir-map`